v2/pkg/templates/templates.go

```diff
@@ -21,6 +21,11 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
+const (
+	// TemplateExtension defines the template default file extension
+	TemplateExtension = ".yaml"
```

Comment:
We can add support for also .yml extension as that can also be used for templates.

Reply:
true, but we can still stick with YAML to avoid mixing up two extensions or to keep it uniform in the template project.